### PR TITLE
Allow global java 2 security enablement in FATs

### PR DESCRIPTION
**Pull request by aguibert**
In order to test all FATs with java 2 security enabled, we would like to be able to specify a property `global.java2.sec=true` which will programatically add `websphere.java.security` to the bootstrap.properties file of all servers (thus enabling java 2 security).

### CODE REVIEW

**Comment by aguibert** [comment]
#build

**Comment by ewhoch** [comment]
There's no information in the PR, is this to turn j2security on during FAT tests always?

**Comment by aguibert** [comment]
@ewhoch updated the PR title and description

File: dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java [hunk 1]
```diff
@@ -1405,54 +1269,12 @@ public void run() {
...
-            }
-        } catch (Exception e) {
-            Log.info(c, method, "Error loading Java 2 Security exempt server list " + e.toString());
-            throw e;
-        } finally {
-            if (br != null) {
-                br.close();
-            }
-        }
+        // Allow servers to opt-out of j2sec by setting
```
Comment (ewhoch) [inline dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java]: This becomes the main logic that turns on j2sec.

**Pull request merged**